consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/ibftmessagedata/CommitPayload.java

```diff
@@ -66,4 +68,37 @@ public Hash getDigest() {
   public Signature getCommitSeal() {
     return commitSeal;
   }
+
+  @Override
+  public ConsensusRoundIdentifier getRoundIdentifier() {
+    return roundIdentifier;
+  }
+
+  @Override
+  public boolean equals(final Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    if (!super.equals(o)) {
+      return false;
+    }
+    final CommitPayload that = (CommitPayload) o;
```

Comment:
has missed the RoundIdentifier, needed for the hashCode and toString (and I assume the other classes?)

Reply:
Just checked the other classes, looks like this was the unfortunate one that I forgot